next.json

```diff
@@ -1200,7 +1200,8 @@
             {"value": 0, "name": "r8 g8 b8 a8 unorm"},
             {"value": 1, "name": "r8 g8 b8 a8 uint"},
             {"value": 2, "name": "b8 g8 r8 a8 unorm"},
-            {"value": 3, "name": "d32 float s8 uint"}
+            {"value": 3, "name": "d32 float s8 uint"},
+            {"value": 4, "name": "r8 unorm"}
```

Comment:
Maybe this could complete the [unorm, uint] x [1, 2, 4 component] matrix.

Reply:
Done.